doc/user_guide/profiling.rst

```diff
@@ -169,11 +169,15 @@ profile_PSyDataInit()
 This method needs to be called once to initialise the profiling tool.
 At this stage this call is not automatically inserted by PSyclone, so
 it is the responsibility of the user to add the call to an appropriate
-location in the application::
+location in the application:
 
-   use profile_psy_data_mod, only : profile_PSyDataInit
-   ...
-   call profile_PSyDataInit()
+.. code-block::
+    :caption: Adding profile_PSyDataInit.
+    :emphasize-lines: 3
```

Comment:
Ooh!

Reply:
The nice things you learn when contributing to a different python project. I also intend to use the mock library (which can potentially simplify tests as well) :)